gossip/emitter/emitter_test.go

```diff
@@ -55,6 +56,12 @@ func TestEmitter(t *testing.T) {
 	external.EXPECT().StateDB().
 		Return(nil).
 		AnyTimes()
+	external.EXPECT().GetLastEvent(idx.Epoch(1), cfg.Validator.ID).
+		Return(nil).
+		Times(1)
+	external.EXPECT().GetHeads(idx.Epoch(1)).
+		Return(hash.Events{(&tdag.TestEvent{}).ID(), (&tdag.TestEvent{}).ID()}).
```

Comment:
nit: no need to specify a return value if this function is supposed to never be called (`Times(0)`)